src/intTest/java/uk/nhs/digital/nhsconnect/lab/results/inbound/NhsAckResponseTest.java

```diff
@@ -0,0 +1,106 @@
+package uk.nhs.digital.nhsconnect.lab.results.inbound;
+
+import lombok.NonNull;
+import org.apache.commons.io.IOUtils;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.io.Resource;
+import org.springframework.test.annotation.DirtiesContext;
+import uk.nhs.digital.nhsconnect.lab.results.IntegrationBaseTest;
+import uk.nhs.digital.nhsconnect.lab.results.mesh.message.MeshMessage;
+import uk.nhs.digital.nhsconnect.lab.results.mesh.message.WorkflowId;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+/**
+ * Tests that the correct NHSACK is returned for valid interchanges and when errors occur in processing.
+ * There is an NHSACK status code for each type of response:
+ * <br>
+ * IAF: Valid Interchange, all Messages accepted
+ * IAP: Valid Interchange, some Messages accepted, some Messages rejected
+ * IRA: Valid Interchange, but all Messages were rejected
+ * IRM: Interchange rejected due to an error in one message
+ * IRI: Interchange rejected due to interchange level error
+ */
+@DirtiesContext
+class NhsAckResponseTest extends IntegrationBaseTest {
+    @Value("classpath:edifact/pathology_IAF.edifact.dat")
+    private Resource edifactIAFResource;
+
+    @Value("classpath:edifact/pathology_IAF_regex.nhsack.dat")
+    private Resource nhsAckIAFResource;
+
+    @Value("classpath:edifact/pathology_IAP.edifact.dat")
+    private Resource edifactIAPResource;
+
+    @Value("classpath:edifact/pathology_IAP_regex.nhsack.dat")
+    private Resource nhsAckIAPResource;
+
+    @Value("classpath:edifact/pathology_IRA.edifact.dat")
+    private Resource edifactIRAResource;
+
+    @Value("classpath:edifact/pathology_IRA_regex.nhsack.dat")
+    private Resource nhsAckIRAResource;
+
+    @Value("classpath:edifact/pathology_IRM.edifact.dat")
+    private Resource edifactIRMResource;
+
+    @Value("classpath:edifact/pathology_IRM_regex.nhsack.dat")
+    private Resource nhsAckIRMResource;
+
+    @Value("classpath:edifact/pathology_IRI.edifact.dat")
+    private Resource edifactIRIResource;
+
+    @Value("classpath:edifact/pathology_IRI_regex.nhsack.dat")
+    private Resource nhsAckIRIResource;
```

Comment:
These are all now unused and can be removed

Reply:
Removed them, thank you!